style.css

```diff
@@ -1,15 +1,343 @@
+  align-items: left;
+  margin: 1.5em 0.5em 0.5em 3.5em;
+}
+.price {
+  display: flex;
+  flex-direction: row;
+  font-size: 2em;
+  font-weight: bold;
+}
+
+.dolar {
+  font-size: 0.7em;
+}
+
+.recipe-title {
+  font-size: 1.5em;
+  text-transform: uppercase;
+  margin-bottom: 0.5em;
+}
+.recipe-text {
+  font-size: 0.75em;
+}
+
+.third-section {
+  display: flex;
+  flex-direction: row;
+  margin-top: 5em;
+}
+
+.meal-photo {
+  display: flex;
+  width: 53%;
+  justify-content: center;
+  margin-left: -1em;
+}
+
+.city {
+  display: flex;
+  flex-direction: column;
+  width: 47%;
+  justify-content: center;
+}
+
+.city-title {
+  color: #0a0a0a;
+  font-size: 2.2em;
+  text-align: left;
+  font-weight: bold;
+  line-height: 1em;
+  padding-left: 25%;
+}
+
+.text-city {
+  font-size: 0.75em;
+  padding-left: 25%;
+  padding-top: 5%;
+
+  margin-right: 6em;
+}
+
+.button-request {
+  margin-left: 25%;
+  margin-top: 2em;
+  border: 1px solid #e31c3a;
+  border-radius: 4px;
+  background-color: #e31c3a;
+  color: #ffffff;
+  outline: none;
+  cursor: pointer;
+  padding: 0.8em;
+  width: 45%;
+  text-transform: uppercase;
+}
+
+footer {
+  margin: 5em 0em;
+  display: flex;
+  flex-direction: column;
+  padding: 0em 28%;
+}
+
+@media only screen and (min-width: 992px) {
+  .mainContainer {
+    width: 60%;
+  }
+}
+
+@media only screen and (min-width: 1368px) {
+  .mainContainer {
+    width: 60%;
+  }
+}
+
+@media only screen and (max-width: 1368px) {
+  .button-contact {
+    margin: 1em 0;
+    border: 1px solid #e31c3a;
+    border-radius: 4px;
+    background-color: #e31c3a;
+    padding: 0.8em;
+    color: #ffffff;
+    outline: none;
+    cursor: pointer;
+    margin-left: 0.26em;
+    text-transform: uppercase;
+    width: 90%;
+  }
+}
+
+@media only screen and (max-width: 1480px) {
+  .third-section {
+    display: flex;
+    flex-direction: column;
+    margin-top: 5em;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .meal-photo {
+    margin: 0em;
+  }
+
+  .city {
+    margin-top: 4em;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+  }
+  .city-title {
+    color: #0a0a0a;
+    font-size: 2.2em;
+    text-align: center;
+    font-weight: bold;
+    line-height: 1em;
+    padding-left: 0%;
+  }
+
+  .text-city {
+    font-size: 0.75em;
+    padding-left: 5em;
+    padding-right: 5em;
+    padding-top: 5%;
+    margin-right: 0em;
+    text-align: center;
+  }
+
+  .button-request {
+    margin-top: 2em;
+    border: 1px solid #e31c3a;
+    border-radius: 4px;
+    background-color: #e31c3a;
+    color: #ffffff;
+    outline: none;
+    cursor: pointer;
+    padding: 0.8em;
+    width: 45%;
+    text-transform: uppercase;
+    margin-left: 0px;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .about-you {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 20%;
```

Comment:
Don't use percent for margin, especially large ones as this. 

 Also, if you are trying to 'push' an element from another - good advice here would be to use the `margin-bottom` on the upper element - follow that implementation everywhere throughout the app